src/In.ProjectEKA.HipService/Discovery/Database/Migrations/20200417080025_RenamingTranscationID.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace In.ProjectEKA.HipService.Discovery.Database.Migrations
+{
+    public partial class RenamingTranscationID : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5386940)